tests/unit/EmojiTest.js

```diff
@@ -147,4 +154,228 @@ describe('EmojiTest', () => {
             ],
         }]);
     });
+
+    describe('update frequently used emojis', () => {
+        let spy;
+
+        beforeAll(() => {
+            Onyx.init({keys: ONYXKEYS});
+            global.fetch = TestHelper.getGlobalFetchMock();
+            spy = jest.spyOn(User, 'updateFrequentlyUsedEmojis');
+        });
+
+        beforeEach(() => {
+            spy.mockClear();
+            Onyx.clear();
+            return waitForPromisesToResolve();
+        });
+
+        it('should put a less frequent and recent used emoji behind', () => {
+            // Given an existing frequently used emojis list with count > 1
+            const frequentlyEmojisList = [
+                {
+                    code: '👋', name: 'wave', count: 2, lastUpdatedAt: 4,
```

Comment:
let's try to mimic real data as much as possible by using unix timestamp in `lastUpdatedAt`?

Reply:
Personally, I think it's fine to have a simple number instead of unix timestamp as it's easier to read. The purpose of `lastUpdatedAt` is to show which emoji is used the most recent and I think as long as the number can represent that clearly, then it's fine.